src/makefile

```diff
@@ -23,7 +23,7 @@ endif
 
 ifdef OPRFHOME
 	OPRFINCDIR=$(OPRFHOME)
-	LDFLAGS+= -Wl,rpath,$(OPRFHOME)
+	LDFLAGS+= -L$(OPRFHOME)
```

Comment:
this we tried already in https://github.com/stef/libopaque/commit/65b15eaf4114f75c2d55b54cdbb6de85a314fd5e

Reply:
maybe I tested incorrectly (no make clean?)? Can't recall, but now it works.